packages/lib/components/shared/side-menu-shared.ts

```diff
@@ -72,6 +73,8 @@ export const renderFolders = (props: Props, renderItem: RenderFolderItem) => {
 
 export const renderTags = (props: Props, renderItem: RenderTagItem) => {
 	const tags = props.tags.slice();
+	const collatorLocale = getCollatorLocale();
+	const collator = getCollator(collatorLocale);
```

Comment:
You could remove even more duplicated code by calling `getCollatorLocale()` inside `getCollator()`. Either as default parameter or inside the function.

Reply:
I agree. getCollatorLocale is used only by getCollator so not really needed

Reply:
I have made it so that `getCollatorLocale()` is passed as the default parameter to `getCollator()`.